locales/en/app.ftl

```diff
@@ -848,6 +848,28 @@ join-firefox = There is a way to protect your privacy. Join {-brand-name}.
 # Link title
 learn-more-link = Learn more.
 
+email-sent = Email Sent!
+
+# Form title
+want-to-add = Want to add another email?
+
+# This is part of a confirmation message that appears after a user has submitted
+# the form to add an additional email to Firefox Monitor.
+verify-the-link = Verify the link sent to { $userEmail } to add it to {-product-name}.
+
+# This is part of a confirmation message that appears after a user has submited the
+# form to add an additional email to Firefox Monitor. { $preferencesLink } is a link
+# to the Preferences page. The code and text for the link is generated elsewhere
+# using the {-preferences} string.
```

Comment:
Is this the `monitor-preferences` string?

Reply:
It's the `preferences` string that we use as a link title in the menu that drops down off the user avatar.

Reply:
My point is that there's not a `-preferences` string